provider/server/server_suite_test.go

```diff
@@ -99,6 +100,11 @@ var _ = BeforeSuite(func() {
 		PathSupportedMachineClasses: machineClassesFile.Name(),
 		RootDir:                     filepath.Join(tempDir, "libvirt-provider"),
 		StreamingAddress:            streamingAddress,
+		Servers: app.ServersOptions{
+			Metrics: app.HTTPServerOptions{
+				Addr: "127.0.0.1:9090",
```

Comment:
Optional: Timeouts can be specified here as well.

Reply:
i will disable metrics server for integration test.

Reply:
fixed